### DISCUSSION TITLE
هیدیفای نصب شده اما وقتی کانفیگ کاربرها را میخوام بریزم روی کلاینتهاشون، خطای کانفیگ میگیرم

### DISCUSSION THREAD

**Discussion by ChakameMortezania** [hiddify/Hiddify-Manager#3608] [Q&A سوالات رایج]
هیدیفای را نصب کردم و دامنه ام هم direct هست اما وقتی میخوام کانفیگ را با QR کد یا روش دیگر روی کلاینت بریزم خطای کانفیگ میگیرم و هیچ کاربری نمیتونه وصل بشه

**Comment by AmirMA-git** [comment]
من همین مشکل رو دارم و نتورک اررور دریافت می کنم

**Comment by AmirMA-git** [comment]
> هیدیفای را نصب کردم و دامنه ام هم direct هست اما وقتی میخوام کانفیگ را با QR کد یا روش دیگر روی کلاینت بریزم خطای کانفیگ میگیرم و هیچ کاربری نمیتونه وصل بشه

مشکلتون حل شده؟؟ لطفا اگر تونستین راه حلی پیدا کنید با من هم به اشتراک بزارید.

**Comment by af60** [comment]
به نظر می رسه مشکل از سمت کلاینت برنامه هست. ولی روی ویندوز ایمپورت انجام میشه
